src/pages/assets/common-css/commonStyles.css

```diff
@@ -0,0 +1,61 @@
+.main-container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 2rem;
+    background-color: var(--background-primary);
+    width: 82.5%;
```

Comment:
As Nikhil has pointed out "At large screens there is white space as the main container does not stretch fully".
Change this width to 100% so it will stretch fully for lager screen as well .